app/controllers/partnering_organization_devise/registrations_controller.rb

```diff
@@ -0,0 +1,60 @@
+class PartneringOrganizationDevise::RegistrationsController < Devise::RegistrationsController
```

Comment:
this should be where all the PO sign up goes:

create, and configure_sign_up_params actions should be uncommented/filled in